commercetools-sdk-base/src/main/java/io/sphere/sdk/models/LocalizedString.java

```diff
@@ -31,6 +31,10 @@
 
     private static final Comparator<Map.Entry<Locale, String>> BY_LOCALE_COMPARATOR = (left, right) -> left.getKey().toString().compareTo(right.getKey().toString());
 
+    public static String getJsonCreatorName(){
```

Comment:
You are right actually i think it needs to be removed, i believe i just forgot to rollback that change.